tests/modules/Makefile

```diff
@@ -26,16 +42,13 @@ TEST_MODULES = \
 
 .PHONY: all
 
-all: $(TEST_MODULES)
-
-32bit:
-	$(MAKE) CFLAGS="-m32" LDFLAGS="-melf_i386"
+all 32bit: $(TEST_MODULES)
```

Comment:
@Phantal this solves the problem of matching test modules build flags with `redis-server` but practically breaks the ability to build a 32-bit redis-server.

Please consider dropping the 32bit target altogether and relying on variable to determine if the user wishes a 32bit or 64bit (default/native) build. The same variable would then apply no matter what target is used, and would also make the `file` workaround unnecessary.

Reply:
I'm not sure I follow how it breaks building 32 bit redis-server.  I was able to build the 32 and 64 bit redis server on linux and solaris as well as the test modules.

Reply:
@Phantal I misread the PR, sorry about that (thought I'm looking at `src/Makefile` for a moment I guess).
Did you consider sourcing `src/.make-settings` which stores the configuration of the last build? I'm not very happy with this mechanism but it's already there and may save the `file` workaround.